app/components/concerns/proposal_detailable.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ProposalDetailable
+  extend ActiveSupport::Concern
+
+  included do
+    def formatted_policy_refs
```

Comment:
I've just moved this method from somewhere else, but would love suggestions for nice ways to refactor it!